src/libaktualizr-c/libaktualizr-c.cc

```diff
@@ -0,0 +1,80 @@
+#include "libaktualizr-c.h"
+
+Aktualizr *Aktualizr_create(const char *config_path) {
+  Aktualizr *a;
+  try {
+    Config cfg(config_path);
+    a = new Aktualizr(cfg);
+  } catch (const std::exception &e) {
+    std::cerr << "Creating exception: " << e.what() << std::endl;
+    return NULL;
+  }
+  return a;
```

Comment:
Maybe, there will be a need/requirement to return some status/error code to am app using the C version of libaktualizr, otherwise the only way to find out what went wrong is by parsing stderr.

Reply:
@mike-sul Absolutely. But first, we'll need to make sure that exceptions thrown by libaktualizr are consistent and meaningful. I plan to work on that next.